imsim/stamp.py

```diff
@@ -99,6 +99,7 @@ def setup(self, config, base, xsize, ysize, ignore, logger):
             base['current_noise_image'] = base['current_image']
             noise_var = galsim.config.CalculateNoiseVariance(base)
             keep_sb_level = np.sqrt(noise_var)/3.
+            self._large_object_sb_level = 3*keep_sb_level
```

Comment:
I guess this implies that this code wasn't unit tested.... Should probably add something for this.